src/ErrorElement.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Button,
+  Flex,
+  Heading,
+  Link as RadixLink,
+  Text,
+} from '@radix-ui/themes'
+import { css } from '@emotion/react'
+
+import GrotCrashed from '@/assets/grot-crashed.svg'
+import { ArrowLeftIcon } from '@radix-ui/react-icons'
+
+const handleCreateIssue = () => {
+  window.studio.browser.openExternalLink(
+    'https://github.com/grafana/k6-studio/issues'
+  )
+}
+
+const handleOpenLogs = () => {
+  window.studio.app.openApplicationLog()
+}
+
+const handleRestart = () => {
+  window.location.replace('/')
+}
```

Comment:
It might be a bit hacky: this essentially reloads the react app, which also erases the zustand stores making sure any erroneous input is reset.

Reply:
I would add this note in the code as a comment as well 👍